source/scripting_v3/GTA.UI/Hud.cs

```diff
@@ -75,6 +76,29 @@ public static bool IsRadarVisible
 			set => Function.Call(Hash.DISPLAY_RADAR, value);
 		}
 
+		/// <summary>
+		/// Locks the minimap at the specified angle relative to the main map layout.
+		/// If a value less than 0 or greater than 360 is used, the minimap angle will become unlocked.
+		/// </summary>
+		/// <param name="angle">The angle in degrees from 0-360 to lock the minimap to.</param>
+		public static void LockRadarDirection(int angle) => Function.Call(Hash.LOCK_MINIMAP_ANGLE, angle);
+
+		/// <summary>
+		/// Unlocks the minimap direction if locked.
+		/// </summary>
+		public static void UnlockRadarDirection() => Function.Call(Hash.UNLOCK_MINIMAP_ANGLE);
+
+		/// <summary>
+		/// Locks the minimap at the specified position in the game world.
+		/// </summary>
+		/// <param name="position">The world coordinates to lock the minimap to.</param>
+		public static void LockRadarPosition(Vector3 position) => Function.Call(Hash.LOCK_MINIMAP_POSITION, position.X, position.Y);
```

Comment:
We could use `Vector2` for this one. `Vector2` and `Vector3` can implicitly convert to each other type like in Unity too in v3 (I think the conversation operator should have been explicit one)...